src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -5589,24 +5638,38 @@ TEST(convolution_gpu, bfyx_iyxo_5x5_fp16)
     auto out_mem = network.get_output("conv_fsv").get_memory();
     cldnn::mem_lock<FLOAT16> out_ptr(out_mem, get_test_stream());
 
-    for (int bi = 0; bi < batch_num; ++bi)
-        for (int fi = 0; fi < output_f; ++fi)
-            for (int yi = 0; yi < output_y; ++yi)
-                for (int xi = 0; xi < output_x; ++xi)
-                {
+    auto output_layout = out_mem->get_layout();
+    if (engine.get_device_info().supports_immad) {
+        // Query oneDNN about recommanded format : src:aBcd16b, wei:ABcd8b8a2b, dst:acdb
+        ASSERT_EQ(output_layout.format, format::byxf);
```

Comment:
then, I think we can just disable this test. The tag is very different from the intention of this test..